### PR TITLE
vxlan: add SerializeTo method

### PR DESCRIPTION
**Pull request by sirspinach**

### CODE REVIEW

**Comment by sirspinach** [comment]
signed

**Comment by gconnell** [comment]
Travis seems to think the code needs to be formatted... could you run 'go fmt'?

https://travis-ci.org/google/gopacket/builds/214010279

File: layers/vxlan.go [hunk 1]
```diff
@@ -61,3 +62,33 @@ func decodeVXLAN(data []byte, p gopacket.PacketBuilder) error {
 	p.AddLayer(vx)
 	return p.NextDecoder(LinkTypeEthernet)
 }
+
+// SerializeTo writes the serialized form of this layer into the
+// SerializationBuffer, implementing gopacket.SerializableLayer.
+// See the docs for gopacket.SerializableLayer for more info.
+func (vx *VXLAN) SerializeTo(b gopacket.SerializeBuffer, opts gopacket.SerializeOptions) error {
+	bytes, err := b.PrependBytes(64)
```
Comment (gconnell) [inline layers/vxlan.go]: Note that this is PrependBYTES, not BITS.  It looks like you're only using 8 bytes here?  I'd expect:  b.PrependBytes(8)

I'll have to look back at the vxlan protocol's frame size to be sure, though...

Comment (sirspinach) [inline layers/vxlan.go]: that's right, it should be 8

**Pull request merged**